go/oasis-test-runner/scenario/e2e/runtime/sentry.go

```diff
@@ -168,16 +165,6 @@ func (s *sentryImpl) Run(ctx context.Context, childEnv *env.Env) error { // noli
 	_, _, _, sentry1P2PPubkey := loadSentryNodeInfo(s.Net.Sentries()[1])
 	_, _, _, sentry2P2PPubkey := loadSentryNodeInfo(s.Net.Sentries()[2])
 
-	compute0 := s.Net.ComputeWorkers()[0]
-	compute0Identity, err := compute0.LoadIdentity()
-	if err != nil {
-		return fmt.Errorf("sentry: error loading compute node identity: %w", err)
-	}
-	// Make sure compute0 has disabled certificate rotation.
-	if !compute0Identity.DoNotRotateTLS {
-		return fmt.Errorf("sentry: compute-0 does not have disabled certificate rotation")
-	}
-
 	compute1 := s.Net.ComputeWorkers()[1]
 	// Query for compute 1 TLS public keys.
```

Comment:
I have no idea what was tested here. I see 2 options:
- Remove this code.
- Test if fetching works.
```
	// Query for compute 1 status.
	compute1Ctrl, err := oasis.NewController(compute1.SocketPath())
	if err != nil {
		return err
	}
	if _, err := compute1Ctrl.GetStatus(ctx); err != nil {
		return fmt.Errorf("failed to get status for compute1: %w", err)
	}
```

Reply:
I removed the code, since the call is also made in other tests.